2020/scoring/clean_and_join_csvs.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/sh
+
+SCORE=../../tools/scoring/score.py
+JOINSCORE=../../tools/scoring/join_results.sh
+COLORDER=../previous-best/unify_column_order.py
```

Comment:
Move this to `../../tools/scoring`?  It's not year specific, right?

Reply:
I think this is year specific.  For instance:
 - this year the fixed solvers were run together with old solvers for sq
 - the solver id 24160 is removed